profiles/migrations/0008_add_first_name_and_last_name_to_profile.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.3 on 2019-10-24 11:26
+
+from django.db import migrations, models
+
+
+def populate_name_from_user_for_existing_profiles(apps, schema_editor):
+    Profile = apps.get_model("profiles", "Profile")
+    for profile in Profile.objects.all():
+        if profile.user is not None:
+            profile.first_name = profile.user.first_name or profile.first_name
+            profile.last_name = profile.user.last_name or profile.last_name
+        profile.save()
```

Comment:
probably want to save only those profiles that were actually modified through the `if` statement?